concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -0,0 +1,331 @@
+Desrochers-2010-330-131	131	botella	bottle	BOTTLE	1321	Noun	6.55882352941176	7	177	6.34
+Desrochers-2010-330-132	132	brisa	breeze			Noun	5	5	102	5.72
+Desrochers-2010-330-133	133	cabeza	head	HEAD	1256	Noun	6.42156862745098	6	1976	6.18
+Desrochers-2010-330-134	134	cabra	goat	GOAT	1502	Noun	5.23529411764706	5	63	5.94
+Desrochers-2010-330-135	135	caída	fall			Noun	5.74509803921569	5	254	6.45
+Desrochers-2010-330-136	136	caldo	broth	SOUP	1547	Noun	5.42156862745098	5	62	5.61
+Desrochers-2010-330-137	137	calle	street	STREET	1362	Noun	6.6078431372549	5	1340	6.8
+Desrochers-2010-330-138	138	cama	bed	BED	1663	Noun	6.73529411764706	4	764	6.41
+Desrochers-2010-330-139	139	cámara	camera	CAMERA	2618	Noun	6.00980392156863	6	223	6.16
+Desrochers-2010-330-140	140	carnet	card	CARD	2619	Noun	6.27450980392157	6	33	6.63
+Desrochers-2010-330-141	141	carta	letter	LETTER	1612	Noun	6.2843137254902	5	472	6.56
+Desrochers-2010-330-142	142	chimenea	chimney	CHIMNEY	1305	Noun	5.29712954133031	8	79	5.27
+Desrochers-2010-330-143	143	choque	shock			Noun	5.76470588235294	6	84	5.9
+Desrochers-2010-330-144	144	cielo	sky	SKY	1732	Noun	6.38235294117647	5	620	6.26
+Desrochers-2010-330-145	145	cigarro	cigar			Noun	6.2156862745098	7	45	5.71
+Desrochers-2010-330-146	146	circuito	circuit			Noun	5.46078431372549	8	106	4.55
+Desrochers-2010-330-147	147	coche	car	CAR	1111	Noun	6.77450980392157	5	688	6.59
+Desrochers-2010-330-148	148	cohete	rocket			Noun	5.11764705882353	6	36	5.65
+Desrochers-2010-330-149	149	corazón	heart	HEART	1223	Noun	6.22549019607843	7	848	6.76
+Desrochers-2010-330-150	150	corcho	cork			Noun	5.31372549019608	6	22	
+Desrochers-2010-330-151	151	costa	coast	COAST	1525	Noun	5.6078431372549	5	276	5.46
+Desrochers-2010-330-152	152	cristal	glass	GLASS	604	Noun	6.31372549019608	7	264	6.61
+Desrochers-2010-330-153	153	cuchillo	knife	KNIFE	1352	Noun	6.51960784313726	8	86	6.31
+Desrochers-2010-330-154	154	cuento	story	STORY	2060	Noun	5.98039215686275	6	275	6.46
+Desrochers-2010-330-155	155	culo	article			Noun	6.35294117647059	4	140	6.05
+Desrochers-2010-330-156	156	diente	tooth	TOOTH	1380	Noun	6.1078431372549	6	39	6.31
+Desrochers-2010-330-157	157	disco	album			Noun	6.34313725490196	5	126	6.09
+Desrochers-2010-330-158	158	documento	document			Noun	5.93137254901961	9	170	5.38
+Desrochers-2010-330-159	159	equipaje	luggage			Noun	5.69607843137255	8	64	5.69
+Desrochers-2010-330-160	160	espalda	back	BACK	1291	Noun	6.35294117647059	7	368	6.21
+Desrochers-2010-330-161	161	fantasma	ghost	GHOST	1175	Noun	5.06862745098039	8	101	4.86
+Desrochers-2010-330-162	162	fiesta	party			Noun	6.6078431372549	6	356	6.7
+Desrochers-2010-330-163	163	folio	page			Noun	6.38235294117647	5	18	6.5
+Desrochers-2010-330-164	164	foto	photo			Noun	6.48039215686275	4	203	6.7
+Desrochers-2010-330-165	165	fruta	fruit	FRUIT	1507	Noun	6.56862745098039	5	71	6.35
+Desrochers-2010-330-166	166	gafas	glasses	GLASSES	2292	Noun	6.49019607843137	5	185	6.66
+Desrochers-2010-330-167	167	gota	gout			Noun	6.01	4	107	6.16
```

Comment:
gota is "Tropfen"

Reply:
It can technically be both, but I'm sure the meaning here is indeed "drop".